orchestra/migrations/0025_iteration.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-04-04 20:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orchestra', '0024_auto_20160325_1916'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Iteration',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('start_datetime', models.DateTimeField(default=django.utils.timezone.now)),
+                ('end_datetime', models.DateTimeField(blank=True, null=True)),
+                ('action', models.IntegerField(blank=True, choices=[(0, 'Submit'), (1, 'Accept'), (2, 'Reject')], null=True)),
+                ('data', jsonfield.fields.JSONField(blank=True, default={})),
+                ('assignment', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='iterations', to='orchestra.TaskAssignment')),
```

Comment:
Update: apparently it will be a required argument for `ForeignKey`s in Django 2.0.